ui/src/components/DenylistTable/index.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { string, shape, func, arrayOf } from 'prop-types';
+import classNames from 'classnames';
+import { pipe, map, sort as rSort } from 'ramda';
+import memoize from 'fast-memoize';
+import { withStyles } from '@material-ui/core/styles';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import LinkIcon from 'mdi-react/LinkIcon';
+import Typography from '@material-ui/core/Typography';
+import { notificationAddress, pageInfo } from '../../utils/prop-types';
+import { VIEW_DENYLIST_PAGE_SIZE } from '../../utils/constants';
+import sort from '../../utils/sort';
+import ConnectionDataTable from '../ConnectionDataTable';
+
+const sorted = pipe(
+  rSort((a, b) => sort(a.node.notificationAddress, b.node.notificationAddress)),
+  map(({ node: { notificationAddress } }) => notificationAddress)
+);
+const tableHeaders = ['Address', 'Type'];
+
+@withStyles(theme => ({
+  tableCell: {
+    textDecoration: 'none',
+  },
+  listItemCell: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '100%',
+    padding: theme.spacing.unit,
+  },
+  listLinkCell: {
+    ...theme.mixins.hover,
```

Comment:
You'll want to add `...theme.mixins.listItemButton,` in addition so that the link icon is faded when doesn't have focus or hover.